docker/importer/volume/surveillance-mediator/endpoint-handler.json

```diff
@@ -127,14 +127,24 @@
     ],
     "response": [
       {
-        "id": "resource-resolver",
+        "id": "resource-resolver-fhir",
         "config": {
           "method": "POST",
           "url": "http://localhost:3003/resource-resolver",
```

Comment:
Perhaps you could change this path as well for consistency

Reply:
addresed